### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by drfarrell**
Added in more graphics, links, and tightened up some language

- [ ] New feature
- [X] Documentation update
- [ ] Bug fix
- [ ] Refactor
- [ ] Release
- [ ] Other

### CODE REVIEW

**Review by Kitenite — COMMENTED** [review]
The images themselves should be links imo https://talk.commonmark.org/t/make-image-act-as-a-hyperlink/2805

File: README.md [hunk 1]
```diff
@@ -58,18 +59,34 @@
...
 * [![React][React.js]][React-url]
 * [![Electron][Electron.js]][Electron-url]
 * [![Tailwind][TailwindCSS]][Tailwind-url]
 * [![Vite][Vite.js]][Vite-url]
 
+## Stay up-to-date
+Onlook officially launched our first version of Onlook on July 08, 2024 and we've shipped a ton since then. Watch releases of this repository to be notified of future updates, and you can follow along with us on [LinkedIn](https://www.linkedin.com/company/onlook-dev) or [Substack](https://onlook.substack.com/) where we write a weekly newsletter. 
+
+
+![Starproject](https://github.com/user-attachments/assets/07742b21-dd98-4be3-b6a6-13d8132af398)
```
Comment (Kitenite) [inline README.md:83]: It's in poor taste to ask for stars imo

File: README.md [hunk 2]
```diff
@@ -140,27 +154,32 @@ The code for the plugins are under `plugins`.
...
+
 If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also [open issues](https://github.com/onlook-dev/onlook/issues).
 
 
 See the [CONTRIBUTING.md](CONTRIBUTING.md) for instructions and code of conduct.
 
 ## Contact
 
+![image](https://github.com/user-attachments/assets/60684b68-1925-4550-8efd-51a1509fc953)
+
```
Comment (Kitenite) [inline README.md:182]: Is there a way to get contributor's avatars on here? I feel like there should be
https://github.com/calcom/cal.com?tab=readme-ov-file#contributors

File: README.md (hunk 1 above)
Comment (drfarrell) [reply README.md:83]: https://github.com/calcom/cal.com?tab=readme-ov-file#stay-up-to-date  👀 

We can remove it I don't mind

**Pull request merged**